docs/DeveloperGuide.md

```diff
@@ -389,144 +363,284 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
 <br>
 
-**Use case 05: Delete a task**
+
+
+**Use case: UC03 - List all tasks**
 
 **MSS**
 
 1. User requests to list tasks.
 
 2. PlaNus shows a list of tasks.
 
-3. User requests to delete a specific task from the list.
-
-4. PlaNus deletes the task.
-
   Use case ends.
 
 **Extensions**
 
-- 2a. The list is empty.
+  - 2a. The list is empty.
+
+    Use case ends.
+
+<br>
+
+**Use case: UC04 - Mark deadlines as done**
+
+**MSS**
+
+1. User requests to list tasks.
+
+2. PlaNus shows a list of tasks.
+
+3. User requests to mark certain deadlines in the list as done.
+
+4. PlaNus marks the deadline as done.
 
   Use case ends.
 
+**Extensions**
+
 - 3a. The given index is invalid.
 
   - 3a1. PlaNus shows an error message.
 
     Use case resumes at step 3.
 
+- 3b. Some indexes given are not of a deadline.
+
+  - 3b1. PlaNus shows an error message.
+
+    Use case resumes at step 3.
+
+- 3c. Deadlines represented by some indexes are already marked as completed.
+
+  - 3c.1 PlaNus shows an error message.
+
+    Use case resumes at step 3.
+
 <br>
 
-**Use case 06: List all tasks**
+**Use case: UC05 - Find tasks**
 
 **MSS**
 
-1. User requests to list tasks.
+1. User <ins>lists a set of tasks (UC03 or UC05)</ins>
```

Comment:
why did you quote UC05 isnt this UC05

Reply:
cuz u can list out a set of tasks using find as well?